tests/mappings/mapping_q_points_real_to_unit_02.output

```diff
@@ -0,0 +1,11 @@
+
+DEAL::Transform on cell with center: -0.08086235035 -0.8200448543 -0.08086235035 with mapping degree 1
+DEAL::Combined transform -0.04896990000 -0.8590530000 -0.04896990000 gives 0.7082205181 0.2510034894 0.7082205181
```

Comment:
This would fail with the previous version of the code, giving a point with unit coordinates around 12.